### ISSUE TITLE
Why Emby?

### ISSUE THREAD

**Issue by yonas** [MediaBrowser/Emby#1253]
Hi, Emby looks very interesting, but I'm wondering what the advantages are for using Emby over Kodi, and vice-versa?

Why was Emby created? For eg. has the Kodi/XBMC source code become too complicated to work with? Is the Kodi team not accepting patches quickly enough?

**Comment by karbowiak** [comment]
Kodi is a media center, Emby is a media server - you can't compare them.

**Comment by yonas** [comment]
Ah, ok. I guess Emby is like Mediatomb circa 2015.

**Comment by Raithlin** [comment]
yonas, Kodi is mainly a front-end - the original concept being that you point it at a hard drive, and it grabs metadata and you play it. Excellent interface, but not as easy to work with when using multiple devices (although this is getting easier with later releases).

Emby is your centralised place for media. From your main server you can connect with almost any device with a client (or the web interface, which is amazing) - even Kodi has a very good add-on. I use Emby to sort and store my media (videos, pics, music, books), and I have Kodi in the living room, and can sync (offline) my phone for videos I want to watch myself, or later (away from my server, for example on a plane).I have friends that use the DLNA component to watch their media using either a PS3, or even DLNA-capable blu-ray players across their local network.

In short, Kodi and Emby work together, not in competition. Emby is quite simply a very capable media server/manager; Kodi an awesome media player.

**Comment by yonas** [comment]
Raithlin, I appreciate the explanation, thanks! I'm going to install Emby on my FreeBSD server :)

**Comment by Raithlin** [comment]
Great stuf!! Be sure to follow the instructions at https://emby.media/downloads/emby-for-kodi/  and not simply grab the (old) addon on Kodi, should you choose to go that route. :)

**Comment by yonas** [comment]
Thanks, I've been using the Emby for Android app on my tablet, and it works great so far.
